x-pack/legacy/plugins/ml/public/data_frame_analytics/common/analytics.ts

```diff
@@ -152,3 +168,62 @@ export const useRefreshAnalyticsList = (
     },
   };
 };
+
+const DEFAULT_SIG_FIGS = 3;
+
+export function getValuesFromResponse(response: RegressionEvaluateResponse) {
+  let meanSquaredError = idx(response, _ => _.regression.mean_squared_error.error) as number;
+  if (meanSquaredError) {
+    meanSquaredError = Number(meanSquaredError.toPrecision(DEFAULT_SIG_FIGS));
+  }
+
+  let rSquared = idx(response, _ => _.regression.r_squared.value) as number;
+  if (rSquared) {
+    rSquared = Number(rSquared.toPrecision(DEFAULT_SIG_FIGS));
+  }
+
+  return { meanSquaredError, rSquared };
+}
+
+export const loadEvalData = async ({
+  isTraining,
+  index,
+  dependentVariable,
+}: {
+  isTraining: boolean;
+  index: string;
+  dependentVariable: string;
+}) => {
+  const results: LoadEvaluateResult = { success: false, eval: null, error: null };
+
+  const config = {
+    index,
+    query: {
+      term: {
+        'ml.is_training': {
+          value: isTraining,
+        },
+      },
+    },
+    evaluation: {
+      regression: {
+        actual_field: dependentVariable,
+        predicted_field: `ml.${dependentVariable}_prediction`,
```

Comment:
can `ml` be customized with `results_field` when the job is created? (it can be customized for outlier jobs)